lib/build-ninja.js

```diff
@@ -114,7 +114,8 @@ export function provideBuilder() {
 }
 
 function parseTarget(line) {
-  const m = /^\s*(\w+): (\w+)$/.exec(line);
+  // For some reason, "line" can have trailing whitespace on Windows.
+  const m = /^\s*([-.@/\\\w]+): (\w+)\s?$/.exec(line);
```

Comment:
Shouldn't it be `\s*` for more robustness?

Reply:
It can be `\s*`. I deliberately did "zero or one" because I don't think `StreamSplitter` should be adding trailing chars to each line anyway. But since it _does_ add chars, might as well just use `\s*`.